.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,16 @@
+on: 
+  push:
+    tags:
+    - '[0-9]+-[0-9]+-[0-9]+.[0-9]+'
+  jobs:
+    deploy-prod:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    steps: 
+    - uses: actions/checkout@v1
+    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+    with:
+      version: '274.0.0'
+      service_account_email: ${{ secrets.GCP_SA_EMAIL_PROD }}
+      service_account_key: ${{ secrets.GCP_SA_KEY_PROD }}
+    - run: gcloud functions deploy ${{ secrets.WEBHOOK_NAME }} --runtime nodejs10 --trigger-http
```

Comment:
Can you label this function, so we can track it's usage and see it broken out in other services.

We've got these set in the [other deploy scripts](https://github.com/mablhq/mail-ingress-processor/blob/master/docker/deploy_cloud_function.sh#L72).

Reply:
Can this also be pinned to the project, like `--project=mabl-dev` so we're sure where things will be deployed?